### PR TITLE
Ref/elastica migration part2

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |N|
|New feature?   |Y|	
|BC breaks?     |N|	
|Deprecations?  |Y|	
|Fixed tickets  |N|	

44 classes with Elasticsearch\Client to go

### CODE REVIEW

File: src/Service/ContentTypeService.php [hunk 1]
```diff
@@ -4,9 +4,13 @@
 
 use Doctrine\Bundle\DoctrineBundle\Registry;
 use Doctrine\ORM\NoResultException;
+use Elastica\Aggregation\Terms;
```
Comment (Davidmattei) [inline src/Service/ContentTypeService.php:7]: Don't like the use of the Elastica namespace in the coreBundle.

We have a search object in CommonBundle add a method, addTerms ?

File: src/Service/DataService.php [hunk 2]
```diff
@@ -12,12 +12,15 @@
 use Doctrine\ORM\OptimisticLockException;
 use Doctrine\ORM\ORMException;
 use Doctrine\ORM\QueryBuilder;
+use Elastica\Query\Term;
```
Comment (Davidmattei) [inline src/Service/DataService.php:15]: Elastica in the coreBundle

**Review by Davidmattei — CHANGES_REQUESTED** [review]
Import off the Elastica\Terms I would not do it, for the rest is a good PR!

File: src/Service/ContentTypeService.php (hunk 1 above)
Comment (theus77) [reply src/Service/ContentTypeService.php]: Fixed

File: src/Service/DataService.php (hunk 2 above)
Comment (theus77) [reply src/Service/DataService.php]: Fixed

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**